packages/framework/aqueduct/src/data-object-factories/pureDataObjectFactory.ts

```diff
@@ -153,30 +199,65 @@ export class PureDataObjectFactory<
 	private readonly registry: IFluidDataStoreRegistry | undefined;
 	private readonly createProps: Omit<CreateDataObjectProps<TObj, I>, "existing" | "context">;
 
+	/**
+	 * {@inheritDoc @fluidframework/runtime-definitions#IFluidDataStoreFactory."type"}
+	 */
+	public readonly type: string;
+
+	/**
+	 * @Remarks Use the props object based constructor instead.
+	 * No new features will be added to this constructor,
+	 * and it will eventually be deprecated and removed.
```

Comment:
Same here re: casing and whether it can be deprecated now
```suggestion
	 * @remarks Use the props object based constructor instead.
	 * No new features will be added to this constructor,
	 * and it will eventually be deprecated and removed.
```